src/couch_replicator_tasks.erl

```diff
@@ -0,0 +1,52 @@
+-module(couch_replicator_tasks).
+
+-export([all/0]).
+
+-include_lib("couch/include/couch_db.hrl").
+-include("couch_replicator.hrl").
+-include("couch_replicator_api_wrap.hrl").
+
+
+all() ->
+    Jobs = ets:tab2list(couch_replicator_scheduler),
+    lists:map(fun(Job) ->
+        Rep = Job#job.rep,
+        Source = case Rep#rep.source of
```

Comment:
Maybe have a local function instead of 2 case statements, have function heads ( `ejson_url` ? )